.github/workflows/set-start-date.yml

```diff
@@ -0,0 +1,84 @@
+name: Set the Start Date in the project
+
+on:
+  issues:
+    types: [assigned]
+jobs:
+  set_date:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Get project data
```

Comment:
In this step, we extract the project data in a JSON file. Later we extract the `Project ID` and the `Date Field ID` of a field named "Start Date" and set them to the env variables named `PROJECT_ID` and `DATE_FIELD_ID`